index.js

```diff
@@ -24,60 +26,73 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  let guess = parseInt(guessInput.value, 10);
+  
 
   hideAllMessages();
 
-  if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
-
-    correctMessage.style.display = '';
-
-    submitButton.disabled = true;
-    guessInput.disabled = true;
-  }
-
-  if (guess !== targetNumber) {
-    if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
-    } else {
-      tooLowMessage.style.display = '';
+  if (guess <= 0) {
+    greaterCero.style.display = '';
+
+  } else if (guess > 99) {
+    lessHundred.style.display = '';
+
+  } else { 
+    attempts += 1;
+    if (guess === targetNumber) {
+      numberOfGuessesMessage.style.display = '';
+      numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
+  
+      correctMessage.style.display = '';
+  
+      submitButton.disabled = true;
+      guessInput.disabled = true;
     }
-
-    const remainingAttempts = maxNumberOfAttempts - attempts;
-
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
+  
+    if (guess !== targetNumber) {
+      if (guess < targetNumber) {
+        tooLowMessage.style.display = '';
+      } else {
+        tooLowMessage.style.display = '';
```

Comment:
When the guess is too high you're still showing the "You guessed too low" message 


https://user-images.githubusercontent.com/9094098/193882949-e0c48705-c674-4fd1-bec1-a7731f07fc41.mov

Reply:
Also, as a bonus/stretch goal, in the video above I showed that it allows you to enter a blank for your guess which will count against me as a guess. It might be nice to give the user a message or warning in that case (or prevent them from entering a blank guess)